content/en/docs/concepts/workloads/_index.md

```diff
@@ -3,5 +3,55 @@ title: "Workloads"
 weight: 50
 description: >
   Understand Pods, the smallest deployable compute object in Kubernetes, and the higher-level abstractions that help you to run them.
+no_list: true
 ---
 
+{{< glossary_definition term_id="workload" length="short" >}}
+Whether your workload is a single component or several that work together, on Kubernetes you run
+it inside a set of [Pods](/docs/concepts/workloads/pods).
+In Kubernetes, a Pod represents a set of running {{< glossary_tooltip text="containers" term_id="container" >}}
+on your cluster.
+
+A Pod has a defined lifecycle. For example, once a Pod is running in your cluster then
+a critical failure on the {{< glossary_tooltip text="node" term_id="node" >}} where that
+Pod is running means that all the Pods on that node fail. Kubernetes treats that level
+of failure as final: you would need to create a new Pod even if the node later recovers.
+
+However, to make life considerably easier, you don't need to manage each Pod directly.
+Instead, you can use _workload resources_ that manage a set of Pods on your behalf.
+These resources configure {{< glossary_tooltip term_id="controller" text="controllers" >}}
+that make sure the right number of the right kind of Pod are running, to match the state
+you specified.
+
+Those workload resources include:
+
+* [Deployment](/docs/concepts/workloads/controllers/deployment/) and [ReplicaSet](/docs/concepts/workloads/controllers/replicaset/)
+  (replacing the legacy resource {{< glossary_tooltip text="ReplicationController" term_id="replication-controller" >}});
+* [StatefulSet](/docs/concepts/workloads/controllers/statefulset/);
+* [DaemonSet](/docs/concepts/workloads/controllers/daemonset/) for running Pods that provide
+  node-local facilities, such as a storage driver or network plugin;
+* [Job](/docs/concepts/workloads/controllers/job/) and
+  [CronJob](/docs/concepts/workloads/controllers/cronjob/)
+  for tasks that run to completion.
+
+There are also two supporting concepts that you might find relevant:
+* [Garbage collection](/docs/concepts/workloads/controllers/garbage-collection/) tidies up objects
+  from your cluster after their _owning resource_ has been removed.
+* There is a [_time-to-live after finished_ controller](/docs/concepts/workloads/controllers/ttlafterfinished/)
```

Comment:
nit: You could remove "There is a"

Reply:
Not sure the grammar would work there. What would the text of the two bullet points be after the revision?

Reply:
[_Time-to-live_ controller](/docs/concepts/workloads/controllers/ttlafterfinished/) removes Jobs once a defined time has passed since the Job completed.

Reply:
Feels like it needs a definite article, to me:
**The** [_Time-to-live_ controller](/docs/concepts/workloads/controllers/ttlafterfinished/) removes Jobs once a defined time has passed since the Job completed.

Reply:
is that better?

Reply:
Sure